openmetadata-ui/src/main/resources/ui/cypress/e2e/Flow/NotificationAlerts.spec.js

```diff
@@ -249,7 +259,111 @@ describe('Notification Alert Flow', () => {
     });
   });
 
-  it('Delete created alert', () => {
+  it('Delete alert with single filter', () => {
+    deleteAlertSteps(ALERT_NAME);
+  });
+
+  it('Create new alert with multiple filters and destinations', () => {
+    verifyResponseStatusCode('@alertsPage', 200);
+
+    cy.get('[data-testid="create-notification"]').click();
+
+    // Enter alert name
+    cy.get('#name').should('be.visible').type(ALERT_NAME);
```

Comment:
no need to add should be visible, instead you can add scrollToView so if its out of the screen cypress will scroll it within the screen